### ISSUE TITLE
[REVIEW]: Rainbow: Automated Air-Liquid Interface Cell Culture Analysis Using Deep Optical Flow

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#4080]
**Submitting author:** @AlphonsG (<a href="http://orcid.org/0000-0002-9607-2839">Alphons Gwatimba</a>)
**Repository:** https://github.com/AlphonsG/Rainbow-Optical-Flow-For-ALI
**Branch with paper.md** (empty if default branch): main
**Version:** v0.1.0
**Editor:** @jmschrei
**Reviewers:** @tpeulen, @Assistedevolution
**Archive:** 10.5281/zenodo.6355080

 

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/f1d97af6910c2dfffc3b1e396b70541f/status.svg)](https://joss.theoj.org/papers/f1d97af6910c2dfffc3b1e396b70541f)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/f1d97af6910c2dfffc3b1e396b70541f"><img src="https://joss.theoj.org/papers/f1d97af6910c2dfffc3b1e396b70541f/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/f1d97af6910c2dfffc3b1e396b70541f/status.svg)](https://joss.theoj.org/papers/f1d97af6910c2dfffc3b1e396b70541f)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/AlphonsG/Rainbow-Optical-Flow-For-ALI" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@tpeulen & @Assistedevolution, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @jmschrei know.

✨ **Please start on your review when you are able, and be sure to complete your review in the next six weeks, at the very latest** ✨ 

## Review checklist for @tpeulen
✨ Important: Please do not use the _Convert to issue_ functionality when working through this checklist, instead, please open any new issues associated with your review <a target="_blank" href="https://github.com/AlphonsG/Rainbow-Optical-Flow-For-ALI">in the software repository associated with the submission</a>. ✨

### Conflict of interest

- [ ] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [ ] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [ ] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/AlphonsG/Rainbow-Optical-Flow-For-ALI">repository url</a>?
- [ ] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [ ] **Contribution and authorship:** Has the submitting author (@AlphonsG) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [ ] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [ ] **Installation:** Does installation proceed as outlined in the documentation?
- [ ] **Functionality:** Have the functional claims of the software been confirmed?
- [ ] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [ ] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [ ] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [ ] **A statement of need:** Does the paper have a section titled 'Statement of Need' that clearly states what problems the software is designed to solve and who the target audience is?
- [ ] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [ ] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [ ] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

## Review checklist for @Assistedevolution
✨ Important: Please do not use the _Convert to issue_ functionality when working through this checklist, instead, please open any new issues associated with your review <a target="_blank" href="https://github.com/AlphonsG/Rainbow-Optical-Flow-For-ALI">in the software repository associated with the submission</a>. ✨

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/AlphonsG/Rainbow-Optical-Flow-For-ALI">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@AlphonsG) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Does the paper have a section titled 'Statement of Need' that clearly states what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @tpeulen, @Assistedevolution it looks like you're currently assigned to review this paper :tada:.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
Wordcount for `paper.md` is 1005

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1007/978-3-030-32583-1_12 is OK
- 10.1088/1367-2630/14/11/115012 is OK
- 10.3389/fcell.2020.00021 is OK
- 10.1038/nmat4357 is OK
- 10.1038/s41467-020-18841-7 is OK
- 10.1038/s41467-020-15164-5 is OK
- 10.1038/s41598-017-17952-4 is OK
- 10.1016/S0065-2458(10)80007-5 is OK

MISSING DOIs

- None

INVALID DOIs

- https://doi.org/10.1111/cea.13097 is INVALID because of 'https://doi.org/' prefix
- https://doi.org/10.1111/rda.13481 is INVALID because of 'https://doi.org/' prefix
```

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.88  T=0.60 s (58.3 files/s, 32756.2 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
HTML                             1           1778              2          13273
Python                          21            366            371           1013
TeX                              1             11              0            198
Jupyter Notebook                 4              0           2042            195
Markdown                         4             91              0            179
YAML                             3             18             48             84
JSON                             1              0              0              6
-------------------------------------------------------------------------------
SUM:                            35           2264           2463          14948
-------------------------------------------------------------------------------


Statistical information for the repository 'a01390c5a2187a51df2b5735' was
gathered on 2022/01/21.
The following historical commit information, by author, was found:

Author                     Commits    Insertions      Deletions    % of changes
Alphons                        111          4434           1058          100.00

Below are the number of rows from each author that have survived and are still
intact in the current revision:

Author                     Rows      Stability          Age       % in comments
Alphons                    1750           39.5          0.9                7.54
```

**Comment by jmschrei** [comment]
Howdy @tpeulen and @Assistedevolution

Thanks for agreeing to review this submission.

The process for conducting a review is outlined above in the checklist from @whedon. Please check the boxes during your review to keep track, as well as make comments in this thread or open issues in the repository itself to point out issues you encounter. Keep in mind that our aim is to improve the submission to the point where it is of high enough quality to be accepted, rather than to provide a yes/no decision, and so having a conversation with the authors is encouraged rather than providing a single review post at the end of the process.

Here are the review guidelines: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html
And here is a checklist, similar to above: https://joss.readthedocs.io/en/latest/review_checklist.html

Please let me know if you encounter any issues or need any help during the review process, and thanks for contributing your time to JOSS and the open source community!

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.04080/joss.04080/10.21105.joss.04080.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.04080/joss.04080/10.21105.joss.04080.pdf) :page_facing_up: :point_left:

**Comment by AlphonsG** [comment]
Hello @jmschrei, just want to confirm if [these](https://github.com/AlphonsG/Rainbow-Optical-Flow-For-ALI/issues/21) suggestions/questions should be there or in this issue page?

**Comment by jmschrei** [comment]
Either way, but it makes more sense to me for issues to be opened in that repo and resolved there, and high level remarks, such as saying that he review is done, are reported here.

**Comment by AlphonsG** [comment]
Alright no problem, thanks.

**Comment by whedon** [comment]
:wave: @Assistedevolution, please update us on how your review is going (this is an automated reminder).

**Comment by whedon** [comment]
:wave: @tpeulen, please update us on how your review is going (this is an automated reminder).

**Comment by Assistedevolution** [comment]
Provided author feedback following review of paper and repository. 
Feedback requested providing clarity around what was being measured and why, More thought suggested into end use of the information. Requested more background (for the new reader) on the problem being solved. 

Flagged in repo fixable package versioning issues with installation under Anaconda, (Note Miniconda is recommended, however I feel its more likely that people will try this with Anaconda).
Waiting on resolution of issues

**Comment by AlphonsG** [comment]
@whedon generate pdf from branch dev2

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch dev2. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.04080/joss.04080/10.21105.joss.04080.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.04080/joss.04080/10.21105.joss.04080.pdf) :page_facing_up: :point_left:

**Comment by Assistedevolution** [comment]
Successfully installed and ran code in the Dev2 branch. 
Successfully Tested both in CLI and GUI mode.
Waiting on:
 -Some notes on documentation
 -Some notes on tests
 -Improvements in readability of paper layout

**Comment by Assistedevolution** [comment]
Checklist complete on Dev2 branch.
Waiting for addition of instructions to run test added to the readme.
And suggestion of breaking up the paper text in smaller paragraphs

**Comment by AlphonsG** [comment]
@whedon generate pdf from branch dev2

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch dev2. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.04080/joss.04080/10.21105.joss.04080.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.04080/joss.04080/10.21105.joss.04080.pdf) :page_facing_up: :point_left:

**Comment by AlphonsG** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1007/978-3-030-32583-1_12 is OK
- 10.1088/1367-2630/14/11/115012 is OK
- 10.3389/fcell.2020.00021 is OK
- 10.1038/nmat4357 is OK
- 10.1038/s41467-020-18841-7 is OK
- 10.1038/s41467-020-15164-5 is OK
- 10.1038/s41598-017-17952-4 is OK
- 10.1016/S0065-2458(10)80007-5 is OK

MISSING DOIs

- None

INVALID DOIs

- https://doi.org/10.1111/cea.13097 is INVALID because of 'https://doi.org/' prefix
- https://doi.org/10.1111/rda.13481 is INVALID because of 'https://doi.org/' prefix
```

**Comment by AlphonsG** [comment]
@whedon check references from branch dev2

**Comment by whedon** [comment]
```
Attempting to check references... from custom branch dev2
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1007/978-3-030-32583-1_12 is OK
- 10.1088/1367-2630/14/11/115012 is OK
- 10.3389/fcell.2020.00021 is OK
- 10.1038/nmat4357 is OK
- 10.1038/s41467-020-18841-7 is OK
- 10.1038/s41467-020-15164-5 is OK
- 10.1111/cea.13097 is OK
- 10.1038/s41598-017-17952-4 is OK
- 10.1016/S0065-2458(10)80007-5 is OK
- 10.1111/rda.13481 is OK
- 10.1038/s41586-020-2649-2 is OK
- 10.1109/MCSE.2007.55 is OK
- 10.25080/Majora-92bf1922-00a is OK
- 10.5281/zenodo.5517697 is OK
- 10.1051/0004-6361/201322068 is OK
- 10.3847/1538-3881/aabc4f is OK
- 10.7717/peerj.453 is OK

MISSING DOIs

- None

INVALID DOIs

- https://doi.org/10.1016/j.patcog.2021.107861 is INVALID because of 'https://doi.org/' prefix
```

**Comment by AlphonsG** [comment]
@whedon check references from branch dev2

**Comment by whedon** [comment]
```
Attempting to check references... from custom branch dev2
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1007/978-3-030-32583-1_12 is OK
- 10.1088/1367-2630/14/11/115012 is OK
- 10.3389/fcell.2020.00021 is OK
- 10.1038/nmat4357 is OK
- 10.1038/s41467-020-18841-7 is OK
- 10.1038/s41467-020-15164-5 is OK
- 10.1111/cea.13097 is OK
- 10.1038/s41598-017-17952-4 is OK
- 10.1016/S0065-2458(10)80007-5 is OK
- 10.1111/rda.13481 is OK
- 10.1038/s41586-020-2649-2 is OK
- 10.1109/MCSE.2007.55 is OK
- 10.25080/Majora-92bf1922-00a is OK
- 10.5281/zenodo.5517697 is OK
- 10.1051/0004-6361/201322068 is OK
- 10.3847/1538-3881/aabc4f is OK
- 10.7717/peerj.453 is OK
- 10.1016/j.patcog.2021.107861 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by Assistedevolution** [comment]
I recommend acceptance of this paper /code based on the dev2 branch

**Comment by AlphonsG** [comment]
Merged dev2 branch with main.

**Comment by AlphonsG** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.04080/joss.04080/10.21105.joss.04080.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.04080/joss.04080/10.21105.joss.04080.pdf) :page_facing_up: :point_left:

**Comment by YuliyaLab** [comment]
Hello @jmschrei,
we completed the revisions suggested by one of the reviewers (suggestions are highly appreciated!), but have not at all heard from the 2nd reviewer. Could we please ask you to check on that? We have been in review since November of last year. 
Many thanks!!

**Comment by jmschrei** [comment]
@tpeulen can you please provide an update on when you'll be able to look at this?

**Comment by tpeulen** [comment]
@AlphonsG I am really sorry for the delay.

**Software:**
I already tried your software before accepting the review and had no issues. You could check all the boxed required for a publication in JOSS. I really liked your software. I only recommend you to additionally provide a conda environment yaml file (to facilitate the installation). Mixing conda and pip did not work very well for me. 

**Manuscript:**
I can only suggest to spend more effort when writing a manuscripts. Frankly, the initial submission was not very well written and it seems that the reviewer @Assistedevolution spent more time for writing the review then the authors spend time when writing their manuscript.

**Overall:**
_Rainbow: Automated Air-Liquid Interface Cell Culture Analysis Using Deep Optical Flow_ (in its current revision) is of sufficient quality for publication in JOSS and I recommend publishing the manuscript.

@jmschrei Somehow I cannot edit the review checklist

**Comment by YuliyaLab** [comment]
Hello @jmschrei,
It seems that we have 2 approvals above. 
Do you require anything form us? 
Thank you,
Yuliya

**Comment by jmschrei** [comment]
@tpeulen we recently switched the backend of our editorial bot (after the start of this review) so I can't re-add you to this review, but I've been told that you can write `@editorialbot generate my checklist`. Will that work for you?

**Comment by YuliyaLab** [comment]
Hello @jmschrei,
we have proofread the manuscript and uploaded the latest version.
It would be great if we could move forward with acceptance.
Thank you,
Yuliya

**Comment by jmschrei** [comment]
@editorialbot generate pdf from branch dev2

**Comment by jmschrei** [comment]
@editorialbot set dev2 as branch

**Comment by jmschrei** [comment]
@editorialbot generate pdf

**Comment by jmschrei** [comment]
@Assistedevolution would you mind checking off the final checkmark in functionality->performance if you've found that to be satisfactory? Then we can move forward with acceptance.

**Comment by AlphonsG** [comment]
> @editorialbot set dev2 as branch

Hello @jmschrei, the branch containing the accepted changes by the reviewers and latest paper is now main, thanks.

**Comment by jmschrei** [comment]
@editorialbot set main as branch

**Comment by jmschrei** [comment]
@editorialbot generate pdf

**Comment by jmschrei** [comment]
Alright, without hearing from either of the reviewers I'm going to go ahead and take steps to accept the paper.

@AlphonsG have you issued a tagged release corresponding to this submission? You should upload it to something like Zenodo and then share the DOI here.

**Comment by jmschrei** [comment]
@editorialbot check references

**Comment by Assistedevolution** [comment]
Have updated the performance checkbox

Can confirm 10.1109/iccv48922.2021.00963  is valid a DOI for Learning to Estimate Hidden Motions with Global Motion Aggregation, the Arxiv reference given in the paper accurate..

In case there was any doubt I propose acceptance of the work

**Comment by AlphonsG** [comment]
> Alright, without hearing from either of the reviewers I'm going to go ahead and take steps to accept the paper.
> 
> @AlphonsG have you issued a tagged release corresponding to this submission? You should upload it to something like Zenodo and then share the DOI here.

Sure, here's the DOI: 10.5281/zenodo.6355080

**Comment by jmschrei** [comment]
Great. Do you have a tagged release on GitHub for this paper?

**Comment by AlphonsG** [comment]
Yeah, this one:

https://github.com/AlphonsG/Rainbow-Optical-Flow-For-ALI/releases/tag/v0.1.0

**Comment by jmschrei** [comment]
@editorialbot set <v0.1.0> as version

**Comment by jmschrei** [comment]
@editorialbot set v0.1.0 as version

**Comment by jmschrei** [comment]
@editorialbot set 10.5281/zenodo.6355080 as archive

**Comment by jmschrei** [comment]
@editorialbot recommend-accept

**Comment by kthyng** [comment]
> Can confirm 10.1109/iccv48922.2021.00963 is valid a DOI for Learning to Estimate Hidden Motions with Global Motion Aggregation, the Arxiv reference given in the paper accurate..

@AlphonsG, @YuliyaLab it seems one of your reviewers is going to the trouble of checking your missing DOIs for you! Can you please fix this now?

**Comment by AlphonsG** [comment]
@editorialbot generate pdf

**Comment by AlphonsG** [comment]
@editorialbot check references

**Comment by AlphonsG** [comment]
> > Can confirm 10.1109/iccv48922.2021.00963 is valid a DOI for Learning to Estimate Hidden Motions with Global Motion Aggregation, the Arxiv reference given in the paper accurate..
> 
> @AlphonsG, @YuliyaLab it seems one of your reviewers is going to the trouble of checking your missing DOIs for you! Can you please fix this now?

Hello, reference fixed now

**Comment by kthyng** [comment]
Hoping to hear back from @tpeulen after a follow up email.

**Comment by tpeulen** [comment]
@editorialbot generate my checklist

**Comment by danielskatz** [comment]
@AlphonsG, @YuliyaLab - As the AEiC on duty this week, I'll now do a final proofread of the paper

**Comment by danielskatz** [comment]
@editorialbot recommend-accept

**Comment by AlphonsG** [comment]
@editorialbot generate pdf

**Comment by AlphonsG** [comment]
Thanks for the suggestions @danielskatz, I've updated the paper accordingly.

**Comment by danielskatz** [comment]
@editorialbot accept

**Comment by danielskatz** [comment]
Congratulations to @AlphonsG (Alphons Gwatimba) and co-authors!!

And thanks to @tpeulen and @Assistedevolution for reviewing, and @jmschrei for editing!
We couldn't do this without you!

**Comment by AlphonsG** [comment]
Thank you @danielskatz and all those involved for your efforts, this is great news!
